src/Shared/Recruit.Vacancies.Client/Infrastructure/QueryStore/IQueryStore.cs

```diff
@@ -14,6 +14,6 @@ public interface IQueryStore
 
         Task<bool> DeleteAsync<T>(string key) where T : QueryProjectionBase;
 
-        Task RefreshAllAsync<T>(IEnumerable<T> items) where T : QueryProjectionBase;
+        Task RecreateAsync<T>(IList<T> items) where T : QueryProjectionBase;
```

Comment:
IList :-o

Reply:
This is to satisfy your resharper informed concerns

Reply:
hmm